crates/aggkit-prover/src/utils/context.rs

```diff
@@ -0,0 +1,82 @@
+use std::collections::HashMap;
```

Comment:
Should we move `Contextualize` trait to `aggchain-proof-types` and implementation to the relevant structure module (e.g. implementation for `AggchainProofInputs` to the `proof.rs`) ?

Reply:
I don't think it is valuable to move this to `aggchain-proof-types` as the `Contextualize` trait is only used for providing `gRPC`'s context for the query. A better place could be in `aggkit-prover-types` as the `Contextualize` trait is using prost's `Bytes`.